pkg/services/govmomi/ipam/parse.go

```diff
@@ -54,7 +54,7 @@ func parseAddressWithPrefix(ipamAddress *ipamv1.IPAddress) (netip.Prefix, error)
 // family must match the other addresses of the same family. IPv4 Gateways are
 // required, but IPv6 gateways are not.
```

Comment:
We should also update the godoc here

```suggestion
// family must match the other addresses of the same family. A gateway address
// is optional. If it is not set this function returns `nil, nil`.
```